tests/test_scanimagetiffimagingextractor.py

```diff
@@ -31,7 +31,7 @@ def file_path():
 @pytest.fixture(scope="module")
 def expected_properties():
     return dict(
-        sampling_frequency=29.1248,
+        sampling_frequency=7.28119,
```

Comment:
This is what is failing on the dev tests. They should fail because we were getting the frame rate instead of the volume rate.

Reply:
Ok, can you link an issue or a PR on neuroconv for this?

Reply:
How could that work?

The released extractor gets the wrong sampling rate. We need to release this first to fix it there I feel.

How you have handled this things on the past? I did not realize that you had downstream dev tests here until now.

Reply:
> The released extractor gets the wrong sampling rate. We need to release this first to fix it there I feel.

Yes, that's right. I just find it easier to coordinate these changes to minimize down-time for neuroconv.